src/ol/webgl/PostProcessingPass.js

```diff
@@ -46,7 +46,7 @@ const DEFAULT_FRAGMENT_SHADER = `
 /**
  * @typedef {Object} UniformInternalDescription
  * @property {import("./Helper").UniformValue} value Value
- * @property {number} location Location
+ * @property {WebGLUniformLocation} location Location
```

Comment:
TypeScript recognized that this type was wrong after removing bind on anonymous function below.